metadata-ingestion/src/datahub/cli/cli_utils.py

```diff
@@ -33,6 +33,15 @@ def first_non_null(ls: List[Optional[str]]) -> Optional[str]:
     return next((el for el in ls if el is not None and el.strip() != ""), None)
 
 
+_T = TypeVar("_T")
+
+
+def value_or(value: Optional[_T], default: _T) -> _T:
```

Comment:
Given that this is similar to `get_or_else` in other languages, we might consider using that name

Reply:
updated